### PR TITLE
fix(env.client): take the base from the api endpoint in sm.json

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/slice-machine/server/src/api/services/getEnv.ts [hunk 1]
```diff
@@ -88,12 +89,12 @@ export default async function getEnv(
...
   const mockConfig = getMockConfig(cwd);
 
-  const client = initClient(
-    cwd,
-    prismicData.value.base,
-    repo,
-    prismicData.value.auth
-  );
+  const base = preferWroomBase(manifestInfo.content.apiEndpoint);
+  const auth = prismicData.value.base.startsWith(base)
```
Comment (Heyrows) [inline packages/slice-machine/server/src/api/services/getEnv.ts]: Good, but I would have done that in the function that retrieve the `prismicData`.
By passing the manifest to it.
That way we ensure that whenever we need the `.prismic` auth informations, we get the manifest first and it is used to know if we're logged in or not.

File: packages/slice-machine/package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
     "build-server": "tsc --project server/tsconfig.json",
     "build-next-app": "cross-env NODE_OPTIONS=--max_old_space_size=8192 next build",
     "export-build": "next export && npm run clean-build",
-    "bundle": "npm run test && npm run build && npm run export-build",
```
Comment (MarcMcIntosh) [inline packages/slice-machine/package.json:31]: are you sure about doing that?

File: packages/slice-machine/server/src/api/services/getPrismicData.ts [hunk 3]
```diff
@@ -17,13 +25,9 @@ export default function getPrismicData(): Result<PrismicData, ErrorWithStatus> {
...
     }
 
-    const authResult = Utils.Cookie.parsePrismicAuthToken(
-      prismicConfig.cookies
-    );
-
     return ok({
       ...prismicData,
-      auth: authResult,
+      auth: Utils.Cookie.parsePrismicAuthToken(prismicConfig.cookies),
```
Comment (MarcMcIntosh) [inline packages/slice-machine/server/src/api/services/getPrismicData.ts:30]: I'm assuming when it throws it'll go to the catch block

**Review by MarcMcIntosh — COMMENTED** [review]
nice

File: packages/slice-machine/server/src/api/services/getPrismicData.ts (hunk 3 above)
Comment (Heyrows) [reply packages/slice-machine/server/src/api/services/getPrismicData.ts:30]: I didn't actually change the code here, just avoided to declare the variable :)

File: packages/slice-machine/package.json (hunk 2 above)
Comment (Heyrows) [reply packages/slice-machine/package.json:31]: I just removed the test part.
I've been thinking about it for a while, running the test or just part of the test when doing is the bundle takes more time for what it provide.
I think we used to do that before having CI.

**Pull request merged**